eclair-core/src/main/scala/fr/acinq/eclair/transactions/Transactions.scala

```diff
@@ -226,28 +226,33 @@ object Transactions {
   /** Fee for an un-trimmed HTLC. */
   def htlcOutputFee(feeratePerKw: FeeratePerKw, commitmentFormat: CommitmentFormat): MilliSatoshi = weight2feeMsat(feeratePerKw, commitmentFormat.htlcOutputWeight)
 
-  /**
-   * While fees are generally computed in Satoshis (since this is the smallest on-chain unit), it may be useful in some
-   * cases to calculate it in MilliSatoshi to avoid rounding issues.
-   * If you are adding multiple fees together for example, you should always add them in MilliSatoshi and then round
-   * down to Satoshi.
-   */
+  /** Fee paid by the commit tx (depends on which HTLCs will be trimmed). */
   def commitTxFeeMsat(dustLimit: Satoshi, spec: CommitmentSpec, commitmentFormat: CommitmentFormat): MilliSatoshi = {
     val trimmedOfferedHtlcs = trimOfferedHtlcs(dustLimit, spec, commitmentFormat)
     val trimmedReceivedHtlcs = trimReceivedHtlcs(dustLimit, spec, commitmentFormat)
     val weight = commitmentFormat.commitWeight + commitmentFormat.htlcOutputWeight * (trimmedOfferedHtlcs.size + trimmedReceivedHtlcs.size)
-    val fee = weight2feeMsat(spec.feeratePerKw, weight)
+    weight2feeMsat(spec.feeratePerKw, weight)
+  }
+
+  /**
+   * While on-chain amounts are generally computed in Satoshis (since this is the smallest on-chain unit), it may be
+   * useful in some cases to calculate it in MilliSatoshi to avoid rounding issues.
+   * If you are adding multiple fees together for example, you should always add them in MilliSatoshi and then round
+   * down to Satoshi.
+   */
+  def toDeduceFromFunderOutputMsat(dustLimit: Satoshi, spec: CommitmentSpec, commitmentFormat: CommitmentFormat): MilliSatoshi = {
```

Comment:
I think I would have gone with `commitTxFeeMsat` and `commitTxTotalCostMsat`
```suggestion
  def commitTxTotalCostMsat(dustLimit: Satoshi, spec: CommitmentSpec, commitmentFormat: CommitmentFormat): MilliSatoshi = {
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1721/commits/856f427fd48ec80e7727c43386af72508a8cc796